### PR TITLE
[MIG] mozaik relation

### PR DESCRIPTION
**Pull request by Olivier-LAURENT**

### CODE REVIEW

File: odoo/addons/mozaik_relation_coordinate/models/res_partner_relation_all.py [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+        string='Created on',
+        readonly=True,
+    )
+    active = fields.Boolean(
+        default=True,
+    )
+
+    @api.model
+    def _get_additional_relation_columns(self):
+        """ adda new columns in SQL view """
```
Comment (RobinetDenisAcsone) [inline odoo/addons/mozaik_relation_coordinate/models/res_partner_relation_all.py]: `adda` -> `add a` missing space

File: odoo/addons/mozaik_relation_coordinate/models/res_partner_relation_all.py [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+    def _correct_vals(self, vals, type_selection):
+        """remove coordinates from values"""
+        vals = super()._correct_vals(vals, type_selection)
+        if type_selection.is_inverse:
+            for key in COORDINATE_FIELDS:
+                vals.pop(key, None)
+        return vals
+
+    @api.onchange('type_selection_id')
+    def onchange_type_selection_id(self):
```
Comment (RobinetDenisAcsone) [inline odoo/addons/mozaik_relation_coordinate/models/res_partner_relation_all.py:123]: `_onchange_type_selection_id` ?

File: odoo/addons/mozaik_relation_coordinate/tests/test_partner_relation_all.py [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+import datetime
+
+from odoo import fields
+from odoo.tests.common import SavepointCase
+
+
+class TestPartnerRelationAll(SavepointCase):
+
+    def setUp(self):
+        super(TestPartnerRelationAll, self).setUp()
```
Comment (RobinetDenisAcsone) [inline odoo/addons/mozaik_relation_coordinate/tests/test_partner_relation_all.py]: can be simplified: `super(TestPartnerRelationAll, self).setUp()` -> `super().setUp()`

File: odoo/addons/mozaik_relation_coordinate/views/res_partner_relation_all.xml [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+                        </page>
+                        <page string="More Info" name="dev" groups="base.group_erp_manager">
+                            <group name="moreinfo">
+                                <field name="is_inverse" />
+                            </group>
+                        </page>
+                    </notebook>
+                </sheet>
+                <!-- module partner_multi_relation is incompatible with mail.thread -->
+                <!--div name="chat" class="oe_chatter">
```
Comment (RobinetDenisAcsone) [inline odoo/addons/mozaik_relation_coordinate/views/res_partner_relation_all.xml]: `<div>` kept for later or can be removed?

File: odoo/addons/mozaik_relation_coordinate/models/res_partner_relation_all.py [hunk 5]
```diff
@@ -0,0 +1,127 @@
+# Copyright 2018 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+import datetime
+
+from odoo import api, fields, models
+
+COORDINATE_FIELDS = [
```
Comment (RobinetDenisAcsone) [inline odoo/addons/mozaik_relation_coordinate/models/res_partner_relation_all.py]: perhaps in a `@api.model` method?

**Review by RobinetDenisAcsone — CHANGES_REQUESTED** [review]
@Olivier-LAURENT

File: odoo/addons/mozaik_relation_coordinate/models/res_partner_relation_all.py (hunk 2 above)
Comment (Olivier-LAURENT) [reply odoo/addons/mozaik_relation_coordinate/models/res_partner_relation_all.py:123]: Native method is unfortunately not underscored !

File: odoo/addons/mozaik_relation_coordinate/views/res_partner_relation_all.xml (hunk 4 above)
Comment (Olivier-LAURENT) [reply odoo/addons/mozaik_relation_coordinate/views/res_partner_relation_all.xml]: I don't no !! It seems it is not a simple detail to inherit this model from `mozaik.abstract.model` or simply `mail.thread`. Thus for now I remove this `<div>`.

**Pull request merged**